### PR TITLE
Initial skeleton

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: quickstart-Azure.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Quickstart Azure DevOps"
+description: "Omg, we cant' believe you're using Microsoft"
```
Comment (fabianhinz) [inline quickstart-Azure.mdx:3]: I get this is an initial skeleton - but - pls don't 🫣

File: mintdump/code.mdx
Comment (fabianhinz) [inline mintdump/code.mdx:1]: i guess everything in `mintdump/` can be removed right?
Screenshot 2023-06-12 at 17 20 03

**Review by fabianhinz — CHANGES_REQUESTED** [review]

File: quickstart-Azure.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+icon: "microsoft"
```
Comment (fabianhinz) [inline quickstart-Azure.mdx:4]: can we use the azure devops logo?

File: mintdump/code.mdx
Comment (ma-zah) [reply mintdump/code.mdx:1]: yeah, it was for me a helping guide for me to edit, but sure we can delete it. How should I do it? Delete and make another pull request?

File: quickstart-Azure.mdx (hunk 1 above)
Comment (ma-zah) [reply quickstart-Azure.mdx:3]: I will change it

**Pull request merged**